parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -132,7 +132,11 @@ where
 		})?;
 
 		// convert fee to MultiAsset
-		let fee = MultiAsset::from((MultiLocation::parent(), fee)).into();
+		let fee = MultiAsset::from((
+			MultiLocation::parent(),
+			fee.base_fee.saturating_add(fee.delivery_fee),
```

Comment:
I would rather we add a helper function `Fees::total`.

```suggestion
			fees.total(),
```

Reply:
[Add method for total fee](https://github.com/Snowfork/snowbridge/pull/975/commits/727288d67a7a073294a1b10122cad6ce02e9f45a)